app/routes.py

```diff
@@ -1,2 +1,103 @@
-from flask import Blueprint
 
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request, abort
+    
+solar_systems_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+
+def valid_int(number,parameter_type):
+    try:
+        int(number)
+    except:
+        abort(make_response({"error": f"{parameter_type} must be an integer"},400)) 
```

Comment:
Style: I'd suggest some slightly different naming to make this clearer to someone using this function:

```suggestion
def check_valid_int(number, parameter_name):
    try:
        int(number)
    except:
        abort(make_response({"error": f"{parameter_name} must be an integer"},400)) 
```